frontend/public/components/chargeback.tsx

```diff
@@ -370,30 +369,9 @@ const ReportsPage_: React.SFC<ReportsPageProps> = props => {
   if (flagPending(props.flags[FLAGS.CHARGEBACK])) {
```

Comment:
i think we can remove this check as well

Reply:
Done.